src/neo/Network/P2P/Payloads/OracleRequestAttribute.cs

```diff
@@ -0,0 +1,13 @@
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class OracleRequestAttribute : TransactionAttribute
```

Comment:
Is this necessary? There is nothing in it. Or is this even possible? If a user call a contract, and the contract use the oracle, there isn't a mechanism to let the wallet to know if it should attach the `OracleRequestAttribute` to the transaction.

Reply:
We need to know that the TX it's a request transaction, the wallet will add this attribute if it detect it

https://github.com/neo-project/neo/pull/1584/files#diff-bac7c9fe6b52541341827f089612cfeaR364-R369

Reply:
If the wallet can detect it, then we can detect it too without attribute, right?

Reply:
But the wallet execute the tx for compute the gas, and it's in this moment when we detect it and flag it, without this "flag" we need to execute the TX in order to detect if it's an oracle request.